CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ CHANGELOG
 
 ---
 
+* 1.0.2 (2023-07-05)
+  * BF: Use type casting for the first parameter of strpos() (PHP Deprecated Construction)
```

Comment:
I believe the deprecation warning is intended to use strict types in the code, not to replace `strpos($` with `strpos(strvar($`